### ISSUE TITLE
Issues when requesting JSON or XML using Doctrine MongoDB ODM

### ISSUE THREAD

**Issue by onema** [FriendsOfSymfony/FOSRestBundle#447]
I have asked this question in StackOverflow before [FOSRestBundle and JMS Serializer, error when geting JSON](http://stackoverflow.com/questions/16173429/fosrestbundle-and-jms-serializer-error-when-geting-json), and there was a [good answer](http://stackoverflow.com/a/16230733/155248) walking me through the correct way to setup the application, which I had before I decided to post the question. I decided to use Doctrine ORM rather than MongoDB ODM and the code worked. 

Basically every time I request json or xml formats I get an error, I don't have the same problem if I request HTML format.

I'm wondering if there may be an issue with the latest version of the FOSRestBundle and the JMSSerializer when using Doctrine MongoDB ODM?

[This](http://d.pr/n/iLLZ) is the full error message I get.
